src/main/java/svc/controllers/SitemapController.java

```diff
@@ -0,0 +1,51 @@
+package svc.controllers;
+
+import javax.inject.Inject;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import svc.managers.SitemapManager;
+import svc.models.Sitemap;
+
+
+@RestController
+@EnableAutoConfiguration
+public class SitemapController {
+	@Inject
+	SitemapManager sitemapManager;
+	
+	@ResponseBody
+	@RequestMapping(
```

Comment:
From https://en.wikipedia.org/wiki/Sitemaps
In the section: Search engine submission
The <sitemap_location> should be the complete URL to the sitemap, such as:
http://www.example.org/sitemap.xml